app/code/Magebit/Faq/Controller/Adminhtml/Question/InlineEdit.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * Magebit_Faq
+ *
+ * @category     Magebit
+ * @package      Magebit_Faq
+ * @author       Artjoms Fomins <info@magebit.com>
+ * @copyright    Copyright (c) 2023 Magebit, Ltd.(https://www.magebit.com/)
+ */
+
+declare(strict_types=1);
+
+namespace Magebit\Faq\Controller\Adminhtml\Question;
+
+use Magento\Backend\App\Action\Context;
+use Magebit\Faq\Api\QuestionRepositoryInterface as QuestionRepository;
+use Magento\Framework\Controller\Result\JsonFactory;
+use Magebit\Faq\Api\Data\QuestionInterface;
+
+/**
+ * Class Controller that responsible for inline editing
+ */
+class InlineEdit extends \Magento\Backend\App\Action
+{
+    /**
+     * Authorization level of a basic admin session
+     *
+     * @see _isAllowed()
+     */
+    public const ADMIN_RESOURCE = 'Magento_Cms::Question';
```

Comment:
If the Magento_Cms::Question is being used, please explain what the "public const ADMIN_RESOURCE" is used for it this situation.